src/imitation/algorithms/preference_comparisons.py

```diff
@@ -954,7 +1141,7 @@ def _training_inner_loop(
             losses.append(output.loss)
             metrics.append(output.metrics)
         losses = th.stack(losses)
-        loss = losses.mean()
+        loss = losses.sum()
```

Comment:
Good catch (otherwise effectively learning rate per member declines with # of members)

Reply:
Should I fix this upstream?

Reply:
I think you should fix this upstream as the change is more relevant there.